charts/kargo/values.yaml

```
@@ -17,9 +17,13 @@ image:
 
 ## @section Global Parameters
 global:
-  ## @param global.clusterSecretsNamespace Indicates a namespace where Secrets associated with cluster-scoped resources can be located.
+  ## @param global.clusterResourcesNamespace Namespace for resources (like Secrets) associated with cluster-scoped resources and for which no cluster-scoped analog exists.
+  clusterResourcesNamespace: kargo-cluster-resources
+  ## @param global.clusterSecretsNamespace **Deprecated in favor of `clusterResourcesNamespace`.** Indicates a namespace where Secrets associated with cluster-scoped resources can be located. Kargo will automatically sync Secrets from one namespace to the other until v1.12.0, at which time, the migration reconciler and support for the clusterSecretsNamespace will both be removed.
   clusterSecretsNamespace: kargo-cluster-secrets
-  ## @param global.createClusterSecretsNamespace Indicates whether the `clusterSecretsNamespace` should be managed by the chart.
+  ## @param global.createClusterResourcesNamespace Indicates whether the `clusterResourcesNamespace` should be managed by the chart.
+  createClusterResourcesNamespace: true
+  ## @param global.createClusterSecretsNamespace **Deprecated in favor of `createClusterResourcesNamespace`.** Indicates whether the `clusterSecretsNamespace` should be managed by the chart.
```

Nit: It's not technically true that this is deprecated in favor of `createClusterResourcesNamespace`. `clusterSecretsNamespace` _is_ deprecated in favor of `clusterResourcesNamespace`, but this field is deprecated because there's no need for it once `clusterSecretsNamespace` is gone.

```suggestion
  ## @param global.createClusterSecretsNamespace **Deprecated, as it is directly related to `clusterSecretsNamespace`, which also is deprecated. See docs for `clusterSecretsNamespace` for more details.** Indicates whether the namespace specified by `clusterSecretsNamespace` should be managed be created when installing the chart.
```